### DISCUSSION TITLE
Performance compare between my setup and the rpi5 with AI hat

### DISCUSSION THREAD

**Discussion by skycryer** [blakeblackshear/frigate#15433] [Ask A Question]
So at the moment I use frigate 14.x on a Mini Pc with N5095 and 32GB Ram with a Coral TPU. I have 5 Streams on Frigate. 

Doorbell - 2k
Children bed 1 - 2k
Children bed 2 - 1080p
Garden Cam - 4k
Garden Cam 2- 1080p

Here is my config if needed
[frigate.txt](https://github.com/user-attachments/files/18079579/frigate.txt)

I saw several times now that if someone moving often between doorbell cam and garden cam the system going up to 100%, not sure why. I use sub streams for detection, added motion masks. But even the coral is sending warning at one point that it is slow. We do not talk about dozen of people tracking just 1-2 person moving between cameras several times.

So my idea is do move frigate to a Raspi5 with this Halo AI hat. Maybe that can also handle compreface then, my Nuc can not. Any experience with performance out yet?

**Comment by NickM-27** [comment]
The best way to see what is using system resources is to use the system metrics page. There is no way to guess what is using resources from a config. 

> Maybe that can also handle compreface then

No, the Pi will be worse off handling compreface in every way.

**Reply by skycryer** [reply]
The system metrics not really showing the problem, not what is using the 100% CPU. Idle the system is on around 35%, but the metric do not show the 65% usage.

**Reply by NickM-27** [reply]
you can use `top` but if it doesn't show then it is probably not frigate. Perhaps it is compreface, in which case I would suggest to you that you increase the required face sizes to run recognition so it does not run so often
